crates/bevy_ecs/src/world/mod.rs

```diff
@@ -444,6 +444,29 @@ impl World {
     /// assert_eq!(world.get::<Position>(entities[0]).unwrap(), &Position { x: 1.0, y: 0.0 });
     /// assert_eq!(world.get::<Position>(entities[1]).unwrap(), &Position { x: 0.0, y: 1.0 });
     /// ```
+    ///
+    /// To iterate over entities in a deterministic order,
+    /// sort the results of the query using the desired component as a key.
+    ///
+    /// ```
+    /// use bevy_ecs::{entity::Entity, world::World};
+    /// let mut world = World::new();
+    /// let a = world.spawn().insert_bundle((2, "abc")).id();
+    /// let b = world.spawn().insert_bundle((3, "xyz")).id();
+    /// let c = world.spawn().insert_bundle((1, "def")).id();
+    /// let mut entities = world.query::<(Entity, &i32, &str)>()
+    ///     .map(|(e, &i &s)| (e, i, &s)) // Copy out of the world
```

Comment:
why do you want to copy out of the world? If you copy (or clone), you would lose the reference to the world and having a `&mut` component in the query wouldn't work

Reply:
It was in the original pre v0.5 example, and I'm not so familiar with Bevy at the moment, still learning it.